ksql-engine/src/test/java/io/confluent/ksql/function/udaf/topkdistinct/IntTopkDistinctKudafTest.java

```diff
@@ -31,81 +34,78 @@
 
 public class IntTopkDistinctKudafTest {
 
-  private Integer[] valueArray;
+  private final List<Integer> valuesArray =
+      ImmutableList.of(10, 30, 45, 10, 50, 60, 20, 60, 80, 35, 25, 60, 80);
   private final TopkDistinctKudaf<Integer> intTopkDistinctKudaf =
       TopKDistinctTestUtils.getTopKDistinctKudaf(3, Schema.INT32_SCHEMA);
 
   @Before
   public void setup() {
-    valueArray = new Integer[]{10, 30, 45, 10, 50, 60, 20, 60, 80, 35, 25,
-                               60, 80};
   }
 
   @Test
   public void shouldAggregateTopK() {
-    Integer[] currentVal = new Integer[]{null, null, null};
-    for (Integer d : valueArray) {
+    List<Integer> currentVal = new ArrayList<>();
+    for (Integer d : valuesArray) {
       currentVal = intTopkDistinctKudaf.aggregate(d, currentVal);
     }
 
-    assertThat("Invalid results.", currentVal, equalTo(new Integer[]{80, 60, 50}));
+    assertThat("Invalid results.", currentVal, equalTo(new ArrayList<Integer>(ImmutableList.of(80,
+                                                                                               60, 50))));
   }
 
   @Test
   public void shouldAggregateTopKWithLessThanKValues() {
-    Integer[] currentVal = new Integer[]{null, null, null};
+    List<Integer> currentVal = new ArrayList<>();
     currentVal = intTopkDistinctKudaf.aggregate(80, currentVal);
 
-    assertThat("Invalid results.", currentVal, equalTo(new Integer[]{80, null, null}));
+    assertThat("Invalid results.", currentVal, equalTo(ImmutableList.of(80)));
   }
 
   @Test
   public void shouldMergeTopK() {
-    Integer[] array1 = new Integer[]{50, 45, 25};
-    Integer[] array2 = new Integer[]{60, 50, 48};
+    List<Integer> array1 = new ArrayList<Integer>(ImmutableList.of(50, 45, 25));
```

Comment:
The merger doesn't modifiy the source aggregates, so there is no reason to duplicate the arrays. This can simply be:

```
List<Integer> array1 = ImmutableList.of(50, 45, 25);
```

There are many instances in this test file and others.

Reply:
Changed all to `ImmutableList`